static/ot/consent/c4394b6a-86ce-428c-a3d3-5b81cabbfed4-test/c4394b6a-86ce-428c-a3d3-5b81cabbfed4-test.json

```diff
@@ -1 +1 @@
","sx","ch","sy","ci","sz","ck","cl","cm","cn","co","cr","tc","td","tf","cu","tg","cv","th","cw","cx","tj","tk","tl","tm","tn","to","tr","tt","tv","tw","dj","tz","dm","do","ua","ug","dz","um","ec","us","eg","eh","uy","uz","va","vc","er","et","ve","vg","vi","vn","vu","fj","fk","fm","fo","wf","ga","gb","ws","gd","ge","gg","gh","gi","gl","gm","gn","gq","gs","gt","gu","gw","gy","xk","hk","hm","hn","ye","ht","id","il","im","in","io","za","iq","ir","zm","je","zw","jm","jo","jp","ke","kg","kh","ki","km","kn","kp","kr","kw","ky","kz","la","lb","lc","lk","lr","ls","ly","ma","md","me","mg","mh","mk","ml","mm","mn","mo","mp","mr","ms","mu","mv","mw","mx","my","mz","na","nc","ne","nf","ng","ni","np","nr","nu","nz","om","pa","pe","pf","pg","ph","pk","pm","pn"],"States":{},"LanguageSwitcherPlaceholder":{"default":"it"},"BannerPushesDown":false,"Default":true,"Global":true,"Type":"GDPR","UseGoogleVendors":false,"VariantEnabled":false,"TestEndTime":null,"Variants":[],"TemplateName":"Siti Prodotti - notice only","Conditions":[],"GCEnable":false,"IsGPPEnabled":false,"EnableJWTAuthForKnownUsers":false}],"IabData":{"cookieVersion":"1","createdTime":"2024-05-31T15:54:20.309000538","updatedTime":"2024-05-31T15:54:20.309010578","cmpId":"28","cmpVersion":"1","consentScreen":"1","consentLanguage":null,"vendorListVersion":0,"maxVendorId":0,"encodingType":"0","globalVendorListUrl":"https://cdn.cookielaw.org/vendorlist/iabData.json"},"IabV2Data":{"cookieVersion":"1","createdTime":"2024-05-31T15:54:20.353960158","updatedTime":"2024-05-31T15:54:20.353969998","cmpId":"28","cmpVersion":"1","consentScreen":"1","consentLanguage":null,"vendorListVersion":0,"maxVendorId":0,"encodingType":"0","globalVendorListUrl":"https://cdn.cookielaw.org/vendorlist/iab2Data.json"},"Iab2V2Data":{"cookieVersion":"1","createdTime":"2024-05-31T15:54:20.398736817","updatedTime":"2024-05-31T15:54:20.398746817","cmpId":"28","cmpVersion":"1","consentScreen":"1","consentLanguage":null,"vendorListVersion":0,"maxVendorId":0,"encodingType":"0","globalVendorListUrl":"https://cdn.cookielaw.org/vendorlist/iab2V2Data.json"},"GoogleData":{"vendorListVersion":1,"googleVendorListUrl":"https://cdn.cookielaw.org/vendorlist/googleData.json"},"ScriptDynamicLoadEnabled":false,"TenantFeatures":{"CookieV2BannerFocus":true,"CookieV2RejectAll":true,"CookieV2GPC":true,"CookieV2AssignTemplateRule":true,"MobileAuthenticatedConsents":true,"CookieV2GeolocationJsonApi":true,"CookieV2GCMDMA":true,"CookieV2TCF21":true,"CookieV2RemoveSettingsIcon":true,"CookieV2BannerLogo":true,"CookieV2GeneralVendors":true,"CookieV2GPP":true},"IsSuppressBanner":false,"IsSuppressPC":false,"PublisherCC":"DE","Domain":"checkout.pagopa.it","TenantGuid":"77f17844-04c3-4969-a11d-462ee77acbe1","EnvId":"app-de-prod","RemoteActionsEnabled":false,"GeoRuleGroupName":"Solo tecnici","GATrackToggle":true,"GATrackAssignedCategory":"AS","WebFormIntegrationEnabled":false,"WebFormSrcUrl":"","WebFormWorkerUrl":"","GppData":{"cmpId":"28"},"AuthenticatedConsent":false}
```

Comment:
why this?

Reply:
It's necessary to make test successful.
One trust js file were imported only for prod env, but not in other envs. In other envs there was simply an nonexistent link that leads tests to fail. You can experiment this failure also in dev or uat env now, simply loading checkout portal and opening console. With this json, we provide an alternative root for onetrust files, but it is a fake configuration, that will make the cookie policy popup not appearing in test envs, but allow the tests to run successfully.